src/js/oTable.js

```diff
@@ -0,0 +1,182 @@
+/* global exports, document, HTMLElement, CustomEvent, setImmediate */
+
+// TODO: Use Dom-Deletgate -- https://www.npmjs.com/package/dom-delegate
+
+function OTable(rootEl) {
+    const sortByColumn = function(columnIndex) {
```

Comment:
Happy to go for that approach, it will stop the function being created each time an OTable instance is instantiated.